### PR TITLE
Week 1 homework

### PR DESCRIPTION
**Pull request by b4nter**

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,21 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.Write("First name:");
+            var firstName = Console.ReadLine();
+            Console.Write("Surname:");
+            var surname = Console.ReadLine();
+            Console.Write("Age:");
+            var age = Console.ReadLine();
```
Comment (caveofjulian) [inline Src/BootCamp.Chapter/Program.cs]: Age is not a string

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,21 @@ class Program
...
+            Console.Write("Weight:");
+            var weight = Convert.ToInt32(Console.ReadLine());
+            Console.Write("Height:");
+            var height = Convert.ToDouble(Console.ReadLine());
+
+            Console.WriteLine(firstName + " " + surname + " is " + age + " years old, his weight is " + weight + " kg and his height is " + height + " cm.");
+            var heightInMeters = height / 100;
+            var bmi = (weight / heightInMeters) / heightInMeters;
+            Console.WriteLine("Body-mass index (BMI) is: " + String.Format("{0:F2}", bmi));
```
Comment (diverges) [inline Src/BootCamp.Chapter/Program.cs]: You still need to calculate BMI for a second person.

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Program.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836806&bid=16063827)

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -6,7 +6,40 @@ class Program
...
+                var height = Convert.ToDouble(Console.ReadLine());
+
+                Console.WriteLine(firstName + " " + surname + " is " + age + " years old, his weight is " + weight + " kg and his height is " + height + " cm.");
+                var heightInMeters = height / 100;
+                var bmi = (weight / heightInMeters) / heightInMeters;
+                Console.WriteLine("Body-mass index (BMI) is: " + String.Format("{0:F2}", bmi));
+
+                Console.Write("\nDo you want to check BMI for another person? y/n: ");
+                var answer = Console.ReadLine();
+                if (answer.ToLower().Equals("y") || answer.ToLower().Equals("yes"))
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836806)

Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836806)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Program.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836806&bid=16063827)

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -6,7 +6,33 @@ class Program
...
+                var height = Convert.ToDouble(Console.ReadLine());
+
+                Console.WriteLine(firstName + " " + surname + " is " + age + " years old, his weight is " + weight + " kg and his height is " + height + " cm.");
+                var heightInMeters = height / 100;
+                var bmi = (weight / heightInMeters) / heightInMeters;
+                Console.WriteLine("Body-mass index (BMI) is: " + String.Format("{0:F2}", bmi));
+
+                Console.Write("\nDo you want to check BMI for another person? y/n: ");
+                var answer = Console.ReadLine().ToLower();
+                doAnotherPerson = answer.Equals("y") || answer.Equals("yes") ? true : false;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836806)

File: Src/BootCamp.Chapter/Program.cs [hunk 5]
```diff
@@ -6,7 +6,33 @@ class Program
...
+                Console.Write("Height:");
+                var height = Convert.ToDouble(Console.ReadLine());
+
+                Console.WriteLine(firstName + " " + surname + " is " + age + " years old, his weight is " + weight + " kg and his height is " + height + " cm.");
+                var heightInMeters = height / 100;
+                var bmi = (weight / heightInMeters) / heightInMeters;
+                Console.WriteLine("Body-mass index (BMI) is: " + String.Format("{0:F2}", bmi));
+
+                Console.Write("\nDo you want to check BMI for another person? y/n: ");
+                var answer = Console.ReadLine().ToLower();
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836806)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Program.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836806&bid=16063827)

File: Src/BootCamp.Chapter/Program.cs [hunk 6]
```diff
@@ -6,7 +6,33 @@ class Program
...
+
+            var doAnotherPerson = true;
+            while (doAnotherPerson)
+            {
+                Console.Write("First name:");
+                var firstName = Console.ReadLine();
+                Console.Write("Surname:");
+                var surname = Console.ReadLine();
+                Console.Write("Age:");
+                //If need to, age can be converted to int.
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Well said :)

File: Src/BootCamp.Chapter/Program.cs [hunk 7]
```diff
@@ -6,7 +6,33 @@ class Program
...
+                //As it is not used as int anywhere, for now leaving it as a string.
+                var age = Console.ReadLine();
+                Console.Write("Weight:");
+                var weight = Convert.ToInt32(Console.ReadLine());
+                Console.Write("Height:");
+                var height = Convert.ToDouble(Console.ReadLine());
+
+                Console.WriteLine(firstName + " " + surname + " is " + age + " years old, his weight is " + weight + " kg and his height is " + height + " cm.");
+                var heightInMeters = height / 100;
+                var bmi = (weight / heightInMeters) / heightInMeters;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: No need brackets

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Program.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836806&bid=16063827)

**Comment by ellmaster** [comment]
image
This still is the wrong branch you want to merge in. :)

**Comment by b4nter** [comment]
wrong pull request, new one is #36 this one will be closed.

**Pull request closed without merging**